.appveyor.yml

```diff
@@ -0,0 +1,133 @@
+version: '{branch}-{build}'
+os: Visual Studio 2015
+clone_folder: C:\projects\oiio
+test: off
+configuration:
+    - Release
+    - Debug
+platform:
+    - x64
+# branches:
+#   only:
+#     - master
+environment:
+  matrix:
+    # - CMAKE_PLATFORM: "Visual Studio 14 2015"
+    #   PYTHON_DIR: "C:\\Python27"
+    - CMAKE_PLATFORM: "Visual Studio 14 2015 Win64"
+      PYTHON_DIR: "C:\\Python27-x64"
+      tbs_arch: "x64"
+      tbs_tools: "msvc14"
+      tbs_static_runtime: 0
+
+install:
+  - cinstall: python
+  - cinst ninja
+  - ps: if($env:build_platform -eq 'x64') {
+          $env:vcvar_arg = 'x86_amd64';
+        }
+        else {
+          $env:vcvar_arg = 'x86';
+        }
+  - call "C:/Program Files (x86)/Microsoft Visual Studio 14.0/VC/vcvarsall.bat"
```

Comment:
Probably not.  It basically depends on whether any tools need to find the compiler `cl` in the path and platform (x86 or x64) environment vars in the environment.  With the visual studio generator cmake seems to find everything automatically (using the registry? or install path searching?  I'm not sure.)